IFC_Engine/Convert/Geometry/FromIFC/Point.cs

```diff
@@ -0,0 +1,43 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Geometry;
+using Xbim.Common.Geometry;
+
+namespace BH.Engine.Adapters.IFC
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /****              Public Methods               ****/
+        /***************************************************/
+
+        public static Point PointFromIFC(this XbimPoint3D point)
+        {
+            return new Point { X = point.X, Y = point.Y, Z = point.Z };
```

Comment:
I don't think these methods need to be in IFC_Engine - they could be in IFC_Adapter, they seem very Adapter specific?